extern/nuklear/nuklear.h

```diff
@@ -19916,6 +19917,22 @@ nk_widget_is_mouse_clicked(struct nk_context *ctx, enum nk_buttons btn)
 }
 
 NK_API int
+nk_widget_is_mouse_click_down(struct nk_context *ctx, enum nk_buttons btn, int down)
```

Comment:
Can you open a PR to nuklear with this change?

Reply:
Yeah I was going to ask about it in the issue, since maybe this should actually be behavior 
 of `nk_widget_has_mouse_click_down` function but since my previous 2 issues still unanswered I was hesitating.